proposals/0024-implicit-resource-binding.md

```diff
@@ -1,4 +1,4 @@
-* Proposal: [NNNN](NNNN-implicit-resource-binding.md)
+* Proposal: [0024](NNNN-implicit-resource-binding.md)
```

Comment:
Did you also want to update the file name too?

Reply:
Yes! Thank you! :)